### DISCUSSION TITLE
Search based on badge criteria

### DISCUSSION THREAD

**Discussion by sgjohnson1981** [community/community#109616] [Repositories]
![Capture](https://github.com/community/community/assets/49596637/b1763cc6-c8b1-4f90-94b2-56ac204a2843)
### Select Topic Area

Question

### Body

Is there a way to search for repos based on the criteria in some of the repo badges? Like if a repo passes build or tests, high percentage test coverage, etc. I was researching training LLMs on permissive source code in the StarCoder paper and it didn't say much about filtering The Stack for code quality. Was curious if models could score higher on the benchmarks if the code in pre-training/fine-tuning was higher quality. I know the badges aren't necessarily a mark of high quality, but was thinking perhaps there's a higher likelihood.

If that's not possible is there a way to convert this to a feature request?

**Comment by ettaboyle** [comment]
Welcome to the GitHub Community, @sgjohnson1981, we're happy you're here! 

You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by sgjohnson1981** [reply]
@ettaboyle I think I posted in "code navigation" or something. Sounded appropriate. And I can't be terribly explicit about my project because it's only started. So far it just entails fine-tuning LLMs for code generation. It's just barely out of the idea phase.

**Comment by github-actions** [comment]
🕒 **Discussion Activity Reminder** 🕒

This Discussion has been labeled as dormant by an automated system for having no activity in the last 60 days. Please consider one the following actions:

1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as `out of date` at the bottom of the page.

2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.

3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.

Note: This dormant notification will only apply to Discussions with the `Question` label. To learn more, see our [recent announcement](https://github.com/orgs/community/discussions/70478).

Thank you for helping bring this Discussion to a resolution! 💬
